megamek/src/megamek/client/ui/swing/GUIPreferences.java

```diff
@@ -312,14 +312,20 @@ protected GUIPreferences() {
 
         store.setDefault(FOV_HIGHLIGHT_RINGS_RADII, "5 10 15 20 25");
         store.setDefault(FOV_HIGHLIGHT_RINGS_COLORS_HSB, "0.3 1.0 1.0 ; 0.45 1.0 1.0 ; 0.6 1.0 1.0 ; 0.75 1.0 1.0 ; 0.9 1.0 1.0 ; 1.05 1.0 1.0 ");
-
+        store.setDefault(FOV_HIGHLIGHT,false);
+        store.setDefault(FOV_HIGHLIGHT_ALPHA, 40);
+        store.setDefault(FOV_DARKEN,true);
+        store.setDefault(FOV_DARKEN_ALPHA, 100);
+        store.setDefault(FOV_STRIPES, 35);
+        store.setDefault(FOV_GRAYSCALE, "false");
```

Comment:
I know this is old, but why is this saved as a string "false"? Any ideas?

Reply:
Hmmmm no :)

Reply:
It's actually treated as a boolean everywhere else, so this is just wrong. I'll correct it.